### PR TITLE
Update responses and error messages in result display

### PR DESCRIPTION
**Pull request by afiqzu**
What title says ^
Also changed 'module' tab name to 'courses' tab

### CODE REVIEW

File: src/main/java/tfifteenfour/clipboard/logic/commands/editcommand/EditSessionCommand.java [hunk 1]
```diff
@@ -21,8 +21,9 @@ public class EditSessionCommand extends EditCommand {
     public static final String COMMAND_TYPE_WORD = "session";
     public static final String MESSAGE_USAGE = COMMAND_WORD
```
Comment (tyx021) [inline src/main/java/tfifteenfour/clipboard/logic/commands/editcommand/EditSessionCommand.java:22]: Should we also add `COMMAND_TYPE_WORD` here?
```suggestion
    public static final String MESSAGE_USAGE = COMMAND_WORD  + " " + COMMAND_TYPE_WORD
```

File: src/main/java/tfifteenfour/clipboard/logic/parser/NewEditCommandParser.java [hunk 2]
```diff
@@ -50,8 +49,8 @@ public EditCommand parse(String args) throws ParseException {
         try {
             addCommandType = CommandTargetType.fromString(ArgumentTokenizer.tokenizeString(args)[1]);
```
Comment (tyx021) [inline src/main/java/tfifteenfour/clipboard/logic/parser/NewEditCommandParser.java:50]: ohno this variable name should be `editCommandType`, my mistake HAHA

**Review by tyx021 — COMMENTED** [review]
Good job 👍

File: src/main/java/tfifteenfour/clipboard/logic/parser/NewEditCommandParser.java (hunk 2 above)
Comment (afiqzu) [reply src/main/java/tfifteenfour/clipboard/logic/parser/NewEditCommandParser.java:50]: Ok!

**Comment by tyx021** [comment]
LGTM :))

**Pull request merged**